lib/matplotlib/_mathtext_data.py

```diff
@@ -1008,8 +1008,8 @@
     'leftparen'                : 40,
     'rightparen'               : 41,
     'bigoplus'                 : 10753,
-    'leftangle'                : 10216,
-    'rightangle'               : 10217,
+    'langle'                   : 10216,
+    'rangle'                   : 10217,
```

Comment:
This seems to collide with https://github.com/matplotlib/matplotlib/blob/515cce40f14a4fe4eed15ddaa569052badb71229/lib/matplotlib/_mathtext_data.py#L467-L468 ?

Reply:
For the `rangle` and `langle` I verified that both entries are the same, should I keep the decimal or the hex version?
(According to #26029, the whole list requires an overhaul for the formatting.)

Reply:
I think the hex is easier to read, but I am indifferent, my concern is just that we do not have two identical entries (very far from each other) in the file.